io.github.voxelcubes.deepqt.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.voxelcubes.deepqt
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: deepqt
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --share=network
+modules:
+  - pypi-dependencies.yaml
+
+  - name: deepqt
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} deepqt
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/c8/e2/5565e3a200efaa6c58d5d40c29f4ea854289c11dca99d286d91ae1031ab1/deepqt-1.1.7-py3-none-any.whl
+        sha256: 37a1821f037dbf887e4609c77725137abebac838dd399377d1f0243c73d538f8
+      - type: git
+        url: https://github.com/VoxelCubes/DeepQt.git
+        commit: ed8a1c51e451f18efab1276bfb0461d176ebac06
+        tag: v1.1.7
+    post-install:
+      - install -Dm644 DeepQt.desktop /app/share/applications/io.github.voxelcubes.deepqt.desktop
+      - install -Dm644 media/deepqt.png /app/share/icons/hicolor/256x256/apps/io.github.voxelcubes.deepqt.png
+      - install -Dm644 flatpak/io.github.voxelcubes.deepqt.appdata.xml /app/share/metainfo/io.github.voxelcubes.deepqt.appdata.xml
+      - desktop-file-edit --set-key="Icon" --set-value=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
+    cleanup:
```

Comment:
cleanup doesn't do what you think it does.

the build directory is removed, there is nothing to cleanup.

Reply:
Oh, good to know, I removed that then, thanks!